WifiART/src/userinterface/ARTMvcView.java

```diff
@@ -0,0 +1,109 @@
+package userinterface;
+
+import structs.ARTSystemState;
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTabbedPane;
+import javax.swing.WindowConstants;
+import java.io.IOException;
+
+public class ARTMvcView extends JFrame {
+    // Height Adjustments
+    // Pixel height not included in rows for tab panel:
+    private static final int TABBED_HEIGHT_OFFSET = 76;
+    // Dimensions (some package-private to allow other windows to adjust automatically)
+    private static final int ROW_WD = 30;
+    private static final int CONTROL_ROWS = 10;
+    private static final int WINDOW_POSITION_X = 10;
+    private static final int WINDOW_POSITION_Y = 10;
+    private static final int FRAME_WIDTH =780;
+    private static final int FRAME_HEIGHT = TABBED_HEIGHT_OFFSET + CONTROL_ROWS * ROW_WD;
+
+    /* Tab name declarations  */
```

Comment:
Use // for the single line comment here, being consistent with the single line comments above.